app/login/page.jsx

```diff
@@ -65,50 +65,91 @@ export default function Login() {
-            Password
-          </label>
-          <button className="w-full text-right text-sm font-semibold text-neutral-700">
-            Forgot password?
-          </button>
-        </div>
-        <div className="mb-12 flex flex-col pb-1 pt-1 text-center">
-          <button
-            onClick={e => handleOnSubmit(e)}
-            className="mb-2 rounded bg-neutral-900 p-4 font-semibold text-neutral-100">
-            Log in
-          </button>
-          <button type="button">
-            <p className="mb-0 mr-2 text-sm font-semibold text-neutral-700">
-              Don&apos;t have an account? Register
-            </p>
-          </button>
-        </div>
-      </form>
-    </main>
-  )
+  const switchToView = newView => {
+    setEmail('')
+    setPassword('')
+    setView(newView)
+  }
+
+   return (
+     <main className="flex min-h-screen flex-col items-center">
+       <div className="pt-20">
+         <LogoBig />
+       </div>
+       <form>
+         <div className="relative mb-4">
+           {view === 'reset-password' ? (
+             // Display only email input field for reset-password view
+             <input
+               onChange={e => setEmail(e.target.value)}
+               type="email"
+               id="email"
+               placeholder="Email"
+               className="mt-16 w-full rounded border-2 border-neutral-400 p-2"
+             />
+           ) : (
+             <>
+               <input
+                 onChange={e => setEmail(e.target.value)}
+                 type="email"
+                 id="email"
+                 placeholder="Email"
+                 className="mt-16 w-full rounded border-2 border-neutral-400 p-2"
+               />
+               {view !== 'check-email' && (
+                 <input
+                   onChange={e => setPassword(e.target.value)}
+                   type="password"
+                   id="password"
+                   placeholder="Password"
+                   className="my-2 w-full rounded border-2 border-neutral-400 p-2"
+                 />
+               )}
+               {/* Render "Forgot password?" link only for the login view */}
+               {view === 'login' && (
+                 <button
+                   type="button"
+                   onClick={() => setView('reset-password')}
+                   className="w-full text-right text-sm font-semibold text-neutral-700">
+                   Forgot password?
+                 </button>
+               )}
+             </>
+           )}
+         </div>
+         <div className="mb-12 flex flex-col pb-1 pt-1 text-center">
+           {view === 'check-email' ? (
+             <p className="mb-0 text-sm font-semibold text-neutral-700">
+               Thank you for registering! Please check your email to confirm.
+             </p>
+           ) : (
+             <>
+               <button
+                 onClick={e => handleOnSubmit(e)}
```

Comment:
this needs to be moved to the form and be an onSubmit event, the form doesn't work properly (eg. if you use enter instead of clicking on this button) without it. Make sure the other buttons doesn't submit the form with event.preventDefault()